src/plugins/GAPPSInstall.js

```diff
@@ -0,0 +1,875 @@
+'use strict';
+
+const OverlayPlugin = require('./util/OverlayPlugin');
+const log = require('loglevel');
+const {progressBar} = require('./util/components');
+log.setDefaultLevel('debug');
+
+// Views definitions
+class InstallingGAPPSView {
+    constructor(plugin) {
+        this.plugin = plugin;
+        this.i18n = plugin.i18n;
+        this.progressBar = null;
+        this.progressPercentage = null;
+        this.progressText = null;
+        const json = {
+            channel: 'systempatcher',
+            messages: ['install opengapps'],
+        };
+        this.plugin.instance.sendEvent(json);
+    }
+
+    render() {
+        const container = document.createElement('div');
+        container.className = 'installing-gapps-view-container';
+
+        // Intro Section (similar to other views)
+        const introSection = document.createElement('div');
+        introSection.className = 'gm-section';
+        const text = document.createElement('div');
+        text.className = 'gm-gapps-text'; // Assuming a similar class for text style
+        text.innerHTML =
+            this.i18n.GAPPS_INSTALL_INTRO_TEXT || 'You are about to install Open GApps on your virtual device.'; // Use appropriate i18n key or default
+        introSection.appendChild(text);
+
+        const separator1 = document.createElement('div');
+        separator1.className = 'gm-separator';
+
+        // Installation Progress Section
+        const progressSection = document.createElement('div');
+        progressSection.className = 'gm-section gm-progress-section'; // Add a class for styling
+
+        const androidIcon = document.createElement('i');
+        androidIcon.className = 'gm-droid-icon'; // Assuming an Android icon class
+
+        this.progressText = document.createElement('div');
+        this.progressText.className = 'gm-progress-text gm-dots-jump-loader';
+        this.progressText.innerHTML = this.i18n.DOWLOADING_TEXT || 'Downloading';
```

Comment:
Not sure "downloading" is the best term: in reality we're uploading the apk to the server.